projects/orquestra-sdk/src/orquestra/sdk/_client/_base/cli/_entry.py

```diff
@@ -150,6 +151,80 @@ def view(wf_run_id: t.Optional[str], config: t.Optional[str]):
     action.on_cmd_call(wf_run_id, config)
 
 
+@workflow.command()
+@cloup.argument(
+    "workflow",
+    required=False,
+    help="""
+May be one of the following:\n
+1) Location of the module where the workflow is defined. Can be a dotted
+name like 'package.module' or a filepath like 'my_proj/workflows.py'. If
+it's a filepath, the project layout is assumed to be "flat-layout" or
+"src-layout" as defined by Setuptools:
+https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#automatic-discovery.\n
+2) The workflow run ID of a previously submitted workflow.
```

Comment:
Suggestion: Having both "workflow" and "id" argument highly complicates stuff where its not really needed. Either leave "workflow" to accept filepath and only that, or remove "id".